File: PerfectSound/PerfectSound.Model/Model/Quote.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PerfectSound.Model.Model
{
     public class Quote
    {
        public int QuoteId { get; set; }
        public int? SongAndPodcastId { get; set; }
        public string QuoteText { get; set; }
    }
}

File: perfect_sound_mobile/lib/models/Quote.dart
import 'package:perfect_sound_mobile/models/SAP.dart';

class Quote {
  final quoteId;
  final int? songAndPodcastId;
  SAP? songAndPodcast;
  final String? quoteText;
  final String? artistName;

  Quote({this.quoteId, this.songAndPodcastId, this.quoteText,this.songAndPodcast,this.artistName});


  factory Quote.fromJson(Map<String, dynamic>json) {

    var songAndPodcast_ =json['songAndPodcast'] != null ? new SAP.fromJson(json['songAndPodcast']) : null;

    return Quote(
        quoteId : ['quoteId'],
        songAndPodcastId : json['songAndPodcastId'],
        songAndPodcast: songAndPodcast_,
        quoteText : json['quoteText'],
        artistName : json['artistName'],
    );
  }
}
